app/assets/stylesheets/application.scss

```diff
@@ -28,3 +24,72 @@
     margin: 0;
   }
 }
+
+// Move down content because we have a fixed navbar that is 50px tall
+body {
+  padding-top: 50px;
+}
+
+//
+// Sidebar
+//
+
+// Hide for mobile, show later
+.sidebar {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  .sidebar {
+    background-color: #f5f5f5;
+    border-right: 1px solid #eee;
```

Comment:
Color literals like `#eee` should only be used in variable declarations; they should be referred to via variable everywhere else.